console/src/telemetry/index.ts

```diff
@@ -0,0 +1,108 @@
+import endpoints from '../Endpoints';
+import globals from '../Globals';
+import { filterEventsBlockList, sanitiseUrl } from './filters';
+import { RUN_TIME_ERROR } from '../components/Main/Actions';
+
+type ReduxAction = {
+  type: string;
+  payload: {
+    pathname: string;
+  };
+  data: any;
```

Comment:
Can we maybe provide a more specific type here or it can be literally anything?

Reply:
It has to be `any` untill we have a type that unites all the redux actions of the app.